components/bsx/Offer/MyOffer.vue

```diff
@@ -34,9 +34,9 @@
       <NeoTableColumn
         v-slot="props"
         field="formatPrice"
-        :label="$t('myOffer.price')"
+        :label="`${$t(`offer.price`)} (${chainSymbol})`"
```

Comment:
Maybe add that as param of translation?

Reply:
meh idk, it's used without chainSymbol in other components